src/components/search-form/__tests__/SearchForm.test.js

```diff
@@ -67,6 +67,23 @@ describe('components/search-form/SearchForm', () => {
         sinon.assert.calledWithMatch(onSubmitSpy, 'cheese', event);
     });
 
+    test('should call onsubmit on search icon submit event', () => {
+        const onSubmitSpy = sinon.spy();
+        const event = {
+            target: {
+                elements: [
+                    {
+                        value: 'cheese',
+                    },
+                ],
+            },
+        };
```

Comment:
Have considered moving this out of the test since it's duplicated, not sure if necessary

Reply:
if its duplicated among the `onSubmit` test then I would consider putting them into the same `describe` then